test/spec/ol/interaction/interaction.test.js

```diff
@@ -87,67 +86,4 @@ describe('ol.interaction.Interaction', function() {
 
   });
 
-  describe('zoomByDelta()', function() {
```

Comment:
It would be good to bring this test coverage back, using functions from the new implementation. Or maybe you have added these tests elsewhere? If so, please point me to them.

Reply:
You will find these tests in the `view.test` file now, including some more:
https://github.com/openlayers/openlayers/blob/7ca32cd40b0d3169e8b8a429446088939d9b5aca/test/spec/ol/view.test.js#L1606-L1713